DevelopmentTools/officialMap/LMapHOCalib.txt

```diff
@@ -0,0 +1,85 @@
+#The valid run modes are: HB, HE, HF, HO
+#The run mode we have right now is: HO
+#The valid run modes are: LMap, EMap, FMap, HT, LMapCalib, EMapCalib
+#The produce mode we have right now is: LMapCalib
+#Constructing HO Calib Object...
+#Dumping HO Calib LMap Object...
+#  Side   Eta   Phi  dPhi  ch_type        name   det      RBX  Sect    RM RM_FI FI_CH ppCol ppRow               ppCpl  ppLC Crate   HTR  fpga   HTR_FI    QIEid   dcc  spigot
```

Comment:
not ch_type, move to Depth. Det should be "CALIB_HO". Remove "name". Please refer to HBHE calib and HF calib map: we do not have "name", but use "subdet" to be "CALIB_HX".